sympy/tensor/indexed_integrals.py

```diff
@@ -0,0 +1,41 @@
+from sympy.tensor import Idx, IndexedBase, Indexed
+from sympy import Integral, Dummy, sympify, Tuple
+
+
+class _IndexedIntegral(Integral):
+    """
+    Experimental class to test integration by indexed variables.
```

Comment:
Even though it is experimental maybe we want some Examples in the docstring? I always found SymPy's larg number of examples to be very useful.